tests/test_request_id.py

```diff
@@ -291,7 +291,7 @@
         {},
         # extra_req_headers
         (
-            ("X-B3-PARENTSPAN", "colossal-edifice",),
+            ("X-B3-PARENTSPANID", "colossal-edifice",),
```

Comment:
Should this test care about case sensitivity? (Guessing probably not...)

Reply:
deliberately not.